test/models/adopter_foster_account_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+class AdopterFosterAccountTest < ActiveSupport::TestCase
+  context "associations" do
```

Comment:
Sorry, I just merged a PR that adds a couple more associations on this model. 
```
has_many :likes, dependent: :destroy
  has_many :liked_pets, through: :likes, source: :pet
```

Can you also add these, please?